core/domain/param_domain.py

```diff
@@ -57,25 +86,58 @@ class ParamChange(object):
     """Value object for a parameter change."""
 
     def __init__(self, name, generator_id, customization_args):
-
+        """Initialize a ParamChange object with the specified arguments.
+
+        Args:
+            name: unicode. The name of the parameter.
+                generator_id: unicode. The type of generator used to create the
```

Comment:
@seanlip I don''t really understand what you meant here. Could you please clarify? Thanks!